### PR TITLE
[backend/frontend] Add "From/To" attributes ref in sightings relationships(#5814)

### LINKED ISSUE #5814: "Sightings" attributes in csv mapper relationships are missing
## Description

In column selection of CSV mapper, sightings attributes are missing (From & To are mandatory fields).

## Environment

1.  OpenCTI version: 5.12.29

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. In data > Processing, create a csvv mapper (+)
2. Add a new representation for relationship
3. Select "sighted in/at" as entity type 

## Expected Output

no bug in relationship

## Actual Output

bug sightings

### PR DESCRIPTION
**Pull request by CelineSebe**
### Proposed changes

* [backend] add a method isStixSightingRelationship
* [frontend] all entities are able to be in a sighting relationship (permisssive option)

### Related issues

* #5814 

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/data/csvMapper/representations/attributes/CsvMapperRepresentationAttributeRefForm.tsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ CsvMapperRepresentationAttributeRefFormProps
 
   // For both entity types, whether they are related-to or revoked-by,
   // we don't need to resolve the different types, as they can link any entity between them.
```
Comment (lndrtrbn) [inline opencti-platform/opencti-front/src/private/components/data/csvMapper/representations/attributes/CsvMapperRepresentationAttributeRefForm.tsx]:
```suggestion
  // For those entity types,
  // we don't need to resolve the different types, as they can link any entity between them.
```

**Review by lndrtrbn — COMMENTED** [review]
So sighting relationships can link every entities between them?

**Comment by CelineSebe** [comment]
> So sighting relationships can link every entities between them?

For now, here and elsewhere on the plateform, can link every entities between them. The notion of sighting is expanded compared to the stix standards

File: opencti-platform/opencti-front/src/private/components/data/csvMapper/representations/attributes/CsvMapperRepresentationAttributeRefForm.tsx (hunk 1 above)
Comment (CelineSebe) [reply opencti-platform/opencti-front/src/private/components/data/csvMapper/representations/attributes/CsvMapperRepresentationAttributeRefForm.tsx]: Thanks !

**Review by lndrtrbn — COMMENTED** [review]
When I try to add a relationship in the CSV mapper I have no attribute listed

image

**Review by lndrtrbn — APPROVED** [review]

**Pull request merged**